Networking/WebSockets/WebSocketImpl.hh

```diff
@@ -45,6 +47,7 @@ namespace litecore::websocket {
             int                 heartbeatSecs;       ///< WebSocket heartbeat interval in seconds (default if 0)
             fleece::alloc_slice networkInterface;    ///< Network interface
             fleece::AllocedDict options;             ///< Other options
+            const C4KeyPair*    externalKey;  ///< Client cert uses external key. Its life is tied to the Replicator.
```

Comment:
Please use `Retained` for safety.

Reply:
I already "retained" in c4RemoteReplicator. It lives with the Replicator.

Reply:
No, it's hard to retain it because class definition of C4KeyPair is completely hidden to CE. To retain, it has to be inside EE ifdef.